docs/administration.landing-zones.md

```diff
@@ -21,17 +21,15 @@ this assignment will remain, but workspace managers can no longer create new pro
 
 Platform Operators can configure a custom Landing Zone [meshTag](./meshstack.metadata-tags.md). The schema is available during the Landing Zone creation and update. For each created Landing Zone different tags can be attached. You can tag or modify tags on a landing zone during creation and edit.
 
-## Include Platform Services as part of the Landing Zone
+## Include Building Blocks as part of the Landing Zone
```

Comment:
Please use lower spelling for workspace, project, building block
This is part of our style guide